### PR TITLE
Creation dialog for challenge

### PR DESCRIPTION
**Pull request by hero101**
### Describe the background of your pull request

What does your pull request do? Does it solve a bug (which one?), add a feature?

### Additional context

Add any other context

### Governance 

- [ ] Documentation is added
- [ ] Test cases are added or updated

By submitting this pull request I confirm that:
- I've read the contributing document https://github.com/alkem-io/.github/blob/master/CONTRIBUTING.md.
- I've read and understand the Code of Conduct https://github.com/alkem-io/.github/blob/master/CODE_OF_CONDUCT.md.
- I understand that any contributions or suggestions I made may make it into the actual code. I've read the License 
     https://github.com/alkem-io/Coordination/blob/master/LICENSE.
- I understand that submitting the pull request requires agreeing to and signing the contributor license agreement.

### CODE REVIEW

File: src/common/components/core/dialog/DialogActions.ts [hunk 1]
```diff
@@ -3,7 +3,10 @@ import MuiDialogActions from '@mui/material/DialogActions';
 
 const DialogActions = styled(MuiDialogActions)(({ theme }) => ({
```
Comment (ccanos) [inline src/common/components/core/dialog/DialogActions.ts:4]: this is used in many places, are you sure about this?

Comment (hero101) [reply src/common/components/core/dialog/DialogActions.ts:4]: I am not sure what are you suggesting.

File: src/domain/shared/utils/useJourneyCreation/useJourneyCreation.ts [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  displayName: string;
+  tagline: string;
+  vision: string;
+  tags: string[];
+}
+
+export const useJourneyCreation = () => {
+  const handleError = useApolloErrorHandler();
+
+  const [createChallengeLazy] = useCreateChallengeMutation({ onError: handleError });
```
Comment (ccanos) [inline src/domain/shared/utils/useJourneyCreation/useJourneyCreation.ts:24]: why lazy? Am I missing something? it's the same createChallenge as always no?

Comment (hero101) [reply src/domain/shared/utils/useJourneyCreation/useJourneyCreation.ts:24]: there is nothing special about this, yes

**Review by ccanos — DISMISSED** [review]

**Review by ccanos — APPROVED** [review]

**Pull request merged**